src/desktop/apps/art_keeps_going/routes.tsx

```diff
@@ -0,0 +1,60 @@
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { stitch } from "@artsy/stitch"
+import { routes } from "reaction/Apps/FeatureAKG/routes"
+// @ts-ignore
+import JSONPage from "../../components/json_page"
+import React from "react"
+import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
+import { Request, Response, NextFunction } from "express"
+
+export const landingPage = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const landing = new JSONPage({ name: "art-keeps-going" })
+    const data = await landing.get()
```

Comment:
Ah, JSON page spits out `snake_case` I imagine. 

Tangential: not that fanatical about `snake_case` but it would be good to add an option to `get({ snakeCase: true })` that then returns an object processed by [camelcase-keys](https://github.com/sindresorhus/camelcase-keys). Could set it to `true` as a default and then manually set it to `false` on our other JSON pages.